src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.cs

```diff
@@ -39,6 +39,24 @@ internal partial class SymbolDisplayVisitor : AbstractSymbolDisplayVisitor
             _lazyAliasMap = aliasMap;
         }
 
+        protected override AbstractSymbolDisplayVisitor MakeFirstSymbolContainingTypeVisitor()
+        {
+            if (!isFirstSymbolVisited || format.KindOptions == SymbolDisplayKindOptions.None)
+            {
+                return this;
+            }
+
+            return new SymbolDisplayVisitor(
+                this.builder,
+                this.format.WithKindOptions(SymbolDisplayKindOptions.None),
```

Comment:
Why the forcing to .None?

Reply:
The kind is already being printed as part of the nested type. We don't want to include the type kind for the containing type or you could end up with a display like this:

```csharp
// namespace X
namespace X {
  // class X.A
  class A {
    // struct class X.A.B
    struct B { }
  }
}
```